pages/api/events/create.js

```diff
@@ -29,5 +29,5 @@ export default async (req, res) => {
   }
```

Comment:
Does line 27 ```res.status(500)``` have to have an `end` as well?

Reply:
good catch